dev-python/dask/dask-1.1.3.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6} )
```

Comment:
Please add here support for Python3.7
Should be `PYTHON_COMPAT=( python2_7 python3_{5,6,7} )`